.env

```diff
@@ -0,0 +1,5 @@
+ZOOKEEPER_VERSION=3.4.14
+DRUID_VERSION=0.16.0-incubating
+OPENRESTY_VERSION=1.15.8.2-1
+# ClusterF chaos proxy version is defined by the corresponding git commit hash id
+CHAOS_PROXY_VERSION=f935b60
```

Comment:
Latest is 39874835 (but that's only a tiny documentation change that doesn't impact this at all).